backend/src/services/worker/worker.service.ts

```diff
@@ -80,6 +81,15 @@ class Worker {
 
     return await this.#workerRepository.create(worker);
   }
+
+  async getAll(): Promise<EAMWorkerGetAllItemResponseDto[]> {
```

Comment:
five a better name for type something like `EAMWorkerResponseDto `

Reply:
```
---
type EAMWorkerGetAllItemResponseDto = {
  id: string;
  name: string;
};

type EAMWorkerGetAllResponseDto = {
  items: EAMWorkerGetAllItemResponseDto[];
};
----

async getAll(): Promise<EAMWorkerGetAllResponseDto> {
   const workers = await this.#workerRepository.getAll();
   return {
      items: workers.map((m) => ({
        id: m.id,
        name: m.name,
      }))
   } 
}

```